project/plugins.sbt

```diff
@@ -1,9 +1,7 @@
 import scala.util.Properties._
 
-val scalaJSVersion = envOrElse("SCALAJS_VERSION", "1.2.0")
-
 addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.6.1")
-addSbtPlugin("org.scala-js" % "sbt-scalajs" % scalaJSVersion)
+addSbtPlugin("org.scala-js" % "sbt-scalajs" % "1.2.0")
 addSbtPlugin("org.scala-native" % "sbt-scala-native" % "0.3.9")
```

Comment:
Scala native is not supported now? Maybe you should drop sbt-scala-native and sbt-scala-native-crossproject plugins?